scripts/registrationStats.coffee

```diff
@@ -21,20 +21,36 @@ filter = (arr, func) ->
   filtered
 
 formatSummary = (data) ->
+  console.log(data)
+
   femAndOther = data.confirmedFemale + data.confirmedOther
   # If every N applicant was male
   minMale = Math.round(100 * femAndOther / data.confirmed)
   # Not considering N applicants
   maxMale = Math.round(100 * femAndOther / (femAndOther + data.confirmedMale))
   nonMale = if minMale isnt maxMale then minMale + '-' + maxMale else minMale
+
+  numSaved = data.demo.gender.M + data.demo.gender.F + data.demo.gender.O + data.demo.gender.N
     
+  percentM_min = Math.round(100 * (data.demo.gender.M ) / numSaved)
+  percentM_max = Math.round(100 * (data.demo.gender.M + data.demo.gender.N) / numSaved)
+  percentF_min = Math.round(100 * data.demo.gender.F / numSaved)
+  percentF_max = Math.round(100 * (data.demo.gender.F + data.demo.gender.N) / numSaved)
+  percentO = Math.round(100 * data.demo.gender.O / numSaved)
+  percentN = Math.round(100 * data.demo.gender.N / numSaved)
+
+
   mit = Math.round(100 * data.confirmedMit / data.confirmed)
```

Comment:
delete unused variable

Reply:
these variables are used in  Saved gender

Reply:
`mit` isn’t used any more, is it?

Reply:
its not currently used, but we wanted to add it back in once confirmed is not 0. Since our goal is to admit about 50% MIT, we want to track Confirmed MIT

Reply:
Ah okay, how about you comment it out and put it by the commented-out percentage breakdown for confirm, then?